docs/content/en/docs/workshops/packages/adot/adot_amp_amg.md

```diff
@@ -0,0 +1,612 @@
+                      - __meta_kubernetes_pod_name
+                      target_label: pod
+                    - action: drop
+                      regex: Pending|Succeeded|Failed|Completed
+                      source_labels:
+                      - __meta_kubernetes_pod_phase
+
+            processors:
+              batch/metrics:
+                timeout: 60s
+
+            exporters:
+              logging:
+                logLevel: info
+              prometheusremotewrite:
+                endpoint: "<AMP-WORKSPACE>/api/v1/remote_write" # Replace with your AMP workspace
+                auth:
+                  authenticator: sigv4auth
+            
+            service:
+              extensions:
+                - health_check
+                - memory_ballast
+                - sigv4auth
+              pipelines:
+                metrics:
+                  receivers: [prometheus]
+                  processors: [batch/metrics]
+                  exporters: [logging, prometheusremotewrite]
+
+      ```
+    </details>
+
+1. Bind additional roles to the service account `pod-identity-webhook` (created at step [Create an EKS-A cluster with IRSA](#create-an-eks-a-cluster-with-irsa)) by applying the following file in the cluster (using `kubectl apply -f <file-name>`). This is because `pod-identity-webhook` by design does not have sufficient permissions to scrap all Kubernetes targets listed in the ADOT config file above. If modifications are made to the Prometheus Receiver, make updates to the file below to add / remove additional permissions before applying the file.
+
+    <details>
+      <summary>Click to expand clusterrole and clusterrolebinding config</summary>
+
+      ```yaml
+      ---
+      apiVersion: rbac.authorization.k8s.io/v1
+      kind: ClusterRole
+      metadata:
+        name: otel-prometheus-role
+      rules:
+        - apiGroups:
+            - ""
+          resources:
+            - nodes
+            - nodes/proxy
+            - services
+            - endpoints
+            - pods
+          verbs:
+            - get
+            - list
+            - watch
+        - apiGroups:
+            - extensions
+          resources:
+            - ingresses
+          verbs:
+            - get
+            - list
+            - watch
+        - nonResourceURLs:
+            - /metrics
+          verbs:
+            - get
+
+      ---
+      apiVersion: rbac.authorization.k8s.io/v1
+      kind: ClusterRoleBinding
+      metadata:
+        name: otel-prometheus-role-binding
+      roleRef:
+        apiGroup: rbac.authorization.k8s.io
+        kind: ClusterRole
+        name: otel-prometheus-role
+      subjects:
+        - kind: ServiceAccount
+          name: pod-identity-webhook  # replace with name of the service account created at step Create an EKS-A cluster with IRSA
+          namespace: default  # replace with namespace where the service account was created at step Create an EKS-A cluster with IRSA
```

Comment:
Not critical, but consider renaming the reference "Create an EKS-A cluster with IRSA" if you rename the heading.

Reply:
Done